demos/device_defender_for_aws/defender_demo.c

```diff
@@ -144,6 +151,17 @@ static uint16_t openUdpPorts[ OPEN_UDP_PORTS_ARRAY_SIZE ];
  */
 static Connection_t establishedConnections[ ESTABLISHED_CONNECTIONS_ARRAY_SIZE ];
 
+/**
+ * @brief Task status array which will store status information of tasks
+ * running in the system, which is used to generate custom metrics.
```

Comment:
(Nit) Suggestion for making the statement clear
```suggestion
 * @brief Array to store status information of tasks running in the system. These will be
 * sent as custom metrics to AWS IoT Device Defender service,
```